frontend/packages/kubevirt-plugin/src/components/ip-addresses.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { getVmiIpAddresses, isVmOff } from 'kubevirt-web-ui-components';
+import { DASH } from '@console/shared';
+import { VMIKind } from '../types';
+
+export const IpAddresses: React.FC<IpAddressesProps> = ({ vmi, vmStatus }) => {
+  const vmIsOff = vmStatus && isVmOff(vmStatus);
+  const ipAddresses = vmIsOff ? [] : getVmiIpAddresses(vmi);
+  return ipAddresses.length > 0 ? ipAddresses.join(', ') : DASH;
+};
+
+type IpAddressesProps = {
+  vmi: VMIKind;
+  vmStatus?: any; // precomputed by higher component for optimization
```

Comment:
we dont have a better type for vmStatus ?

Reply:
done